include/odp/api/spec/traffic_mngr.h

```diff
@@ -189,6 +190,63 @@ extern "C" {
  * tree/hierarchy of nodes.
  */
 
+/**
+ * TM queue specific statistics counters
+ */
+typedef struct odp_tm_queue_stats_t {
+	/** Number of octets in successfully transmitted packets. In case of
+	 *  Ethernet, packet size includes MAC header. */
+	uint64_t octets;
+
+	/** Number of successfully transmitted packets. */
+	uint64_t packets;
+
+	/** Number of octets in packets discarded due to other reasons (e.g.
+	 * aging) than errors. */
+	uint64_t discard_octets;
+
+	/** Number of packets discarded due to other reasons (e.g. aging) than
+	 *  errors. */
+	uint64_t discard_packets;
```

Comment:
This could be simply "discards" as in other APIs. If other APIs add discard_octets later, we have the same logic in all APIs (discards + discard_octets)

Reply:
Fixed in V5